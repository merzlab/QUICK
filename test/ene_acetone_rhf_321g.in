HF BASIS=3-21G cutoff=1.0e-9 denserms=1.0e-6 ENERGY DIPOLE CHARGE=0

 C     0.00000000   1.31405400  -0.77434500
 C     0.00000000  -1.31405400  -0.77434500
 C     0.00000000   0.00000000   0.00000000
 O     0.00000000   0.00000000   1.21582500
 H     0.00000000   1.18206200  -1.85902200
 H     0.00000000  -1.18206200  -1.85902200
 H    -0.87985900   1.89899800  -0.48320100
 H    -0.87985900  -1.89899800  -0.48320100
 H     0.87985900   1.89899800  -0.48320100
 H     0.87985900  -1.89899800  -0.48320100


