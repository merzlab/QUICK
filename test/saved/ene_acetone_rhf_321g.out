  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Wed Mar 17 11:46:47 2021
| INPUT FILE :    ene_acetone_rhf_321g.in
| OUTPUT FILE:    ene_acetone_rhf_321g.out
| DATE FILE  :    ene_acetone_rhf_321g.dat
| BASIS SET PATH: /scratch/cruzeiro/quick/basis/

 @ Read Job And Atom

  KEYWORD=HF BASIS=3-21G CUTOFF=1.0E-9 DENSERMS=1.0E-6 ENERGY DIPOLE CHARGE=0                                                                                                                                     
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 DIPOLE
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = 3-21G,  TYPE = CARTESIAN
| BASIS FILE = /scratch/cruzeiro/quick/basis//3-21G.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = C 
 BASIS FUNCTIONS =    9
 For Atom Kind =    2
 ELEMENT = O 
 BASIS FUNCTIONS =    9
 For Atom Kind =    3
 ELEMENT = H 
 BASIS FUNCTIONS =    2

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =   10    NUMBER OF ATOM TYPES        =    3
 NUMBER OF HYDROGEN ATOM =    6    NUMBER OF NON-HYDROGEN ATOM =    4
 NUMBER OF ELECTRONS     =   32

 -- INPUT GEOMETRY -- :
    C           0.0000       1.3141      -0.7743
    C           0.0000      -1.3141      -0.7743
    C           0.0000       0.0000       0.0000
    O           0.0000       0.0000       1.2158
    H           0.0000       1.1821      -1.8590
    H           0.0000      -1.1821      -1.8590
    H          -0.8799       1.8990      -0.4832
    H          -0.8799      -1.8990      -0.4832
    H           0.8799       1.8990      -0.4832
    H           0.8799      -1.8990      -0.4832

 -- DISTANCE MATRIX -- :
              1           2           3           4           5
      1     0.00000
      2     4.96640     0.00000
      3     2.88228     2.88228     0.00000
      4     4.50672     4.50672     2.29758     0.00000
      5     2.06486     5.14308     4.16308     6.22519     0.00000
      6     5.14308     2.06486     4.16308     6.22519     4.46755
      7     2.07102     6.31932     4.05910     5.09422     3.37041
      8     6.31932     2.07102     4.05910     5.09422     6.58969
      9     2.07102     6.31932     4.05910     5.09422     3.37041
     10     6.31932     2.07102     4.05910     5.09422     6.58969
              6           7           8           9          10
      6     0.00000
      7     6.58969     0.00000
      8     3.37041     7.17717     0.00000
      9     6.58969     3.32539     7.91012     0.00000
     10     3.37041     7.91012     3.32539     7.17717     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   48
 NSHELL =   24 NPRIM  =   78
 JSHELL =   24 JBASIS =   42


 @ End Reading Molecular Information 


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1   -192.260244120      ------         0.157     1        0.01      0.14      0.01    0.2075E+01  0.6148E-01  0.6450E+00
|  2   -190.750476378  -.150977E+01       0.168     2        0.01      0.16      0.01    0.2762E+00  0.1994E-01  0.1601E+00
|  3   -190.837578128  0.871018E-01       0.169     3        0.01      0.16      0.01    0.1686E+00  0.1105E-01  0.1142E+00
|  4   -190.877722106  0.401440E-01       0.168     4        0.01      0.16      0.01    0.3363E-01  0.3367E-02  0.2094E-01
|  5   -190.881708029  0.398592E-02       0.169     5        0.01      0.16      0.01    0.7652E-02  0.1214E-02  0.1090E-01
|  6   -190.882104765  0.396736E-03       0.159     6        0.01      0.15      0.01    0.3620E-02  0.4462E-03  0.2548E-02
|  7   -190.882178744  0.739784E-04       0.139     7        0.01      0.13      0.01    0.1380E-02  0.2209E-03  0.1771E-02
|  8   -190.882193716  0.149719E-04       0.139     8        0.01      0.13      0.01    0.6355E-03  0.9097E-04  0.9455E-03
|  9   -190.882195947  0.223114E-05       0.138     9        0.01      0.13      0.01    0.3095E-03  0.3973E-04  0.4682E-03
| 10   -190.882196385  0.438377E-06       0.139    10        0.01      0.13      0.01    0.1342E-03  0.2497E-04  0.3002E-03
| 11   -190.882196501  0.115831E-06       0.139     1        0.01      0.13      0.01    0.2727E-04  0.7083E-05  0.7708E-04
| 12   -190.882196510  0.897643E-08       0.139     2        0.01      0.13      0.01    0.6333E-05  0.1623E-05  0.1453E-04
| 13   -190.882196511  0.561499E-09       0.147     3        0.02      0.13      0.01    0.2561E-05  0.4692E-06  0.2877E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  13 CYLCES
| MAX ERROR = 0.256124E-05   RMS CHANGE = 0.469243E-06   MAX CHANGE = 0.287679E-05
| -----------------------------------------------
 ELECTRONIC ENERGY    =  -309.632913210
 CORE_CORE REPULSION  =   118.750716700
 TOTAL ENERGY         =  -190.882196511

 @ End Energy calculation


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   C             -0.6813            -0.3201
   C             -0.6813            -0.3201
   C              0.5062             0.2309
   O             -0.5582            -0.2734
   H              0.2084             0.0969
   H              0.2084             0.0969
   H              0.2495             0.1222
   H              0.2495             0.1222
   H              0.2495             0.1222
   H              0.2495             0.1222
   TOTAL        -0.0000            -0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    0.0000   -0.0000   -2.9665    2.9665

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     0.395558000( 16.03%)
| TOTAL SCF TIME      =     2.026570000( 82.11%)
|       TOTAL OP TIME      =     1.862286000( 75.46%)
|             TOTAL 1e TIME      =     0.055904000(  2.27%)
|             TOTAL 2e TIME      =     1.743015000( 70.62%)
|       TOTAL DII TIME      =     0.159631000(  6.47%)
|             TOTAL DIAG TIME    =     0.078923000(  3.20%)
| TOTAL TIME          =     2.468063000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  2.5 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Wed Mar 17 11:46:48 2021
